app/helpers/sessions_helper.rb

```diff
@@ -19,6 +19,12 @@ def must_be_logged_in
     end
   end
 
+  def cannot_go_home
```

Comment:
For now I agree, but I was actually thinking we should just change the root_path of this site to the posts_path. If the user is not logged in they are redirected to the homepage with the log in/sign in anyway